src/utilcode/loaderheap.cpp

```diff
@@ -723,6 +729,9 @@ struct LoaderHeapFreeBlock
             }
 #endif
 
+#if LOADER_HEAP_REDZONE_VALUE != 0
```

Comment:
It means that zero-filling is not necessary for GlobalLoaderHeap. Am I right? I though that it is a part of LoaderHeap semantics.